commands/update.py

```diff
@@ -87,7 +88,7 @@ def do_update(args):
             logging.info("Waiting a bit for things to settle")
             time.sleep(9)
             (version, board_id) = board.identify(args.root)
-            if version == new_version:
+            if version == new_version or (args.commit_hash and new_version in version[-10:]):
```

Comment:
Is there a simpler expression that will work for both? Like I'm not particularly concerned about "version must exactly equal new_version" so if this is just "if new_version is anywhere in version" I'm OK with that if it works in both cases.

Reply:
I think that should work but not 100% certain. I'll try it out tonight and try to make sure it won't lead to any false positives.